Readme.md

```diff
@@ -1269,43 +35,36 @@ service / on gmailEventListener {
  export packagePAT=<Personal access token>
  ```
 
-## Building the Source
+### Building the source
 
-To clone the repository: Clone this repository using the following command:
-```
-git clone https://github.com/ballerina-platform/module-ballerinax-googleapis.gmail.git
-```
-### Building java libraries
-To build java libraries execute the following command.
-```
- ./gradlew clean build
- ````
-
-### Build ballerina connector.
+Execute the commands below to build from the source.
 
-Execute the commands below to build from the source after installing Ballerina Swan Lake Alpha5.
+1. To build Java dependency
+    ```
+    ./gradlew clean build
+    ```
 
-1. To build the library:
-```shell script
-    bal build -c ./gmail
-```
+2. * To build the package:
+        ```shell script
+            bal build -c ./gmail
+        ```
 
-2. To build the module without the tests:
-```shell script
-    bal build -c --skip-tests ./gmail
-```
+   * To build the package without tests:
+        ```shell script
+            bal build -c --skip-tests ./gmail
+        ```
 
-# Contributing to Ballerina
+## Contributing to Ballerina
 
 As an open source project, Ballerina welcomes contributions from the community. 
 
 For more information, go to the [contribution guidelines](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md).
 
-# Code of Conduct
+## Code of conduct
 
 All the contributors are encouraged to read the [Ballerina Code of Conduct](https://ballerina.io/code-of-conduct).
```

Comment:
Use same sentence as mentioned in  the guideline docs.

Reply:
Fixed by ddd90073069938a3660747b110c4643c53e57b4f